org.linux_hardware.hw-probe.yaml

```diff
@@ -0,0 +1,393 @@
+app-id: org.linux_hardware.hw-probe
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: 1.6
```

Comment:
Why not using 18.08?

Reply:
With 18.08 I've got the following error when building smartmontools:

    GNU Automake 1.10 (up to 1.15) is required to bootstrap smartmontools from SVN

How to fix this?

Reply:
Sorry, no idea.

Reply:
@bilelmoussaoui 1.16.1

```
$ flatpak run --command=automake org.freedesktop.Sdk//18.08 --version

automake (GNU automake) 1.16.1
Copyright (C) 2018 Free Software Foundation, Inc.
License GPLv2+: GNU GPL version 2 or later <https://gnu.org/licenses/gpl-2.0.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by Tom Tromey <tromey@redhat.com>
       and Alexandre Duret-Lutz <adl@gnu.org>.
```

Reply:
I've added `automake-1.15` to `/app` as a module, but now build of libusb is failed with the error:

    Can't locate Automake/Config.pm in @INC (you may need to install the Automake::Config module).

Reply:
This is fixed by one of the previous commits.